pkg/common/log/log.go

```diff
@@ -1,18 +1,25 @@
 package log
 
 import (
+	"fmt"
 	"io"
 	"os"
 
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	DefaultFormat = ""
+	JSONFormat    = "JSON"
+	TextFormat    = "TEXT"
+)
+
 type Logger struct {
 	*logrus.Logger
 	io.Closer
 }
 
-func NewLogger(logLevel string, fileName string) (*Logger, error) {
+func NewLogger(logLevel, format, fileName string) (*Logger, error) {
```

Comment:
Maybe we document here that `logLevel` is a logrus Level https://godoc.org/github.com/sirupsen/logrus#Level ?

And instead of `fileName` we call that arg `outputFile` ?

Reply:
> Maybe we document here that logLevel is a logrus Level 

These are actually SPIRE log levels, defined as constants above. We convert them to logrus levels in a switch statement below.

> And instead of fileName we call that arg outputFile ?

Done!